src/tsickle.ts

```diff
@@ -476,7 +476,18 @@ abstract class ClosureRewriter extends Rewriter {
         // But it's fine to translate TS "implements Class" into Closure
         // "@extends {Class}" because this is just a type hint.
         const typeChecker = this.typeChecker;
-        const sym = this.mustGetSymbolAtLocation(impl.expression);
+        const sym = this.typeChecker.getSymbolAtLocation(impl.expression);
+        if (!sym) {
+          // It's possible for a class declaration to extend an expression that
+          // is not a symbol, for example when a mixin function is used to build
+          // a base class, as in `declare MyClass extends MyMixin(MyBaseClass)`.
+          //
+          // Handling this correctly is tricky. Closure also doesn't handle this
```

Comment:
Can you expand on how Closure doesn't handle the extends syntax? Does it crash, does it report errors?

Reply:
Done (it throws, added issue link).